Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -230,7 +230,6 @@ - (void)createSimulatorWithContext:(BPExecutionContext *)context {
     };
 
     handler.onSuccess = ^{
-        context.simulatorCreated = YES;
```

Comment:
Why remove this? It's wasting cycles if we try to delete a simulator that was never created. Are you seeing something weird like partially-created simulators even though success is never reported? I read your note about how you tested, but I don't understand what the original issue actually is.